### PR TITLE
Translation of translation guide into Korean. :)

### PR DESCRIPTION
**Pull request by cwdoh**
This is an experimental translation with GitLocalize. I've reviewed this by myself.
Glad to see easier process to translate WF docs. Thanks :)

cc. @agektmr, @PaulKinlan

[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/ko/review/275)

### CODE REVIEW

**Comment by agektmr** [comment]
For the new translation process, I would like to make sure to have someone native to a language to review a translation.

I would like someone who speaks Korean to review this. Doing a review here or on GitLocalize (they have such feature) is an open question, so I would like to hear your opinion @cwdoh . Let's try both options and see which works better.

For this PR, can we assign @techhtml to review?

**Comment by techhtml** [comment]
Sure :) I will review this translation as soon as possible.

**Comment by cwdoh** [comment]
@agektmr Sure! I'm waiting for your review, @techhtml. :)

File: src/content/ko/resources/translations.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+현재 [GitLocalize](https://gitlocalize.com/)라는 도구를 통한 번역을 실험하고 있습니다. 번역을 통한 기여의 시작
+방법은 다음 단계를 따라주시기 바랍니다.
+
+1. [GitLocalize의 Web Fundamentals 저장소](https://gitlocalize.com/repo/107)로 이동합니다.
+2. 여러분의 GitHub 계정으로 로그인합니다.
+3. 번역하고자 하는 문서를 검색합니다.
+4. 번역을 시작합니다.
+5. 번역이 완료되면 리뷰를 위해 번역된 글을 전송합니다.
```
Comment (techhtml) [inline src/content/ko/resources/translations.md]: 문장의 순서를 바꾸면 좋을 거 같습니다.

Suggest:
리뷰를 위해 번역이 완료되면 번역된 글을 전송합니다

Comment (cwdoh) [reply src/content/ko/resources/translations.md]: Done.

**Comment by cwdoh** [comment]
@agektmr 
Basically, I would like to have review comments on here. Footprints in PR histories will help of other people to make better translations. And reviews of knits like typos would be okay on GitLocalize, I believe that it makes good data to enhance their ML translation or can be used for good examples of translations of technical materials. Hope keeping our eyes on some more use-cases for better options. :)

**Comment by agektmr** [comment]
@cwdoh Thanks for the feedback. Let's keep on using GitHub side reviews for the moment and see if it works.

**Pull request merged**